pep-0702.rst

```diff
@@ -0,0 +1,277 @@
+PEP: 702
+Title: Typing support for deprecations
+Author: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Content-Type: text/x-rst
+Created: 30-Dec-2022
+Python-Version: 3.12
+
+
+Abstract
+========
+
+This PEP adds an ``@typing.deprecated()`` decorator that marks a class or function
+as deprecated, enabling static checkers to warn when it is used.
```

Comment:
```suggestion
as deprecated, enabling static checkers to produce a warning when it is used.
```

Reply:
I like the more concise current wording better